### PR TITLE
Add readme_renderer support

### LINKED ISSUE #27: test .md files on valid markdown
see https://github.com/hpcugent/vsc-install/pull/23#issuecomment-181424889

### PR DESCRIPTION
**Pull request by stdweird**
fixes #27

### CODE REVIEW

**Comment by stdweird** [comment]
test this please

@JensTimmerman first failure as expected, installed via
```
export PYTHONUSERBASE=/var/lib/jenkins/python_deps
export PYTHONPATH=/var/lib/jenkins/workspace/vsc-install-prbuilder/PYTHON/CPython-2.7/lib:/var/lib/jenkins/workspace/vsc-install-prbuilder/PYTHON/CPython-2.7:/var/lib/jenkins/.vsc/lib/python
/var/lib/jenkins/.local/bin/python2.7 `which easy_install` --user readme_renderer
```

**Comment by boegel** [comment]
@stdweird: see description in #23: `python setup.py check -rs`

**Comment by stdweird** [comment]
@boegel i'm aware of the command to run. but when i run it on, i get following error

```
...
INFO: running check
WARN: warning: Check: :49: (ERROR/3) Unexpected indentation.

WARN: warning: Check: Invalid markup which will not be rendered on PyPI.

error: Please correct your package.
```
just like the tests. i wonder why you didn't see this

**Comment by boegel** [comment]
@stdweird: hmm, I remember seeing that, and I thought I fixed it... Apparently not. :(

**Comment by stdweird** [comment]
@boegel can you retest current master just to make sure this not some dependency issue? also any tips on how to fix it are appreciated. i can't get it working without removing the indentation altogether.

**Comment by boegel** [comment]
I remember fiddling with it too, I think it's a false negative...

Also, I'm getting this now (worse than what you're getting), not sure what's going on. :(

```
INFO: running check
WARN: warning: check: Unexpected indentation. (line 49)

WARN: warning: check: Inline literal start-string without end-string. (line 47)

WARN: warning: check: Inline interpreted text or phrase reference start-string without end-string. (line 47)

WARN: warning: check: Block quote ends without a blank line; unexpected unindent. (line 50)

WARN: warning: check: Unexpected indentation. (line 52)

WARN: warning: check: Block quote ends without a blank line; unexpected unindent. (line 53)

WARN: warning: check: Possible title underline, too short for the title.
Treating it as ordinary text because it's so short. (line 54)

WARN: warning: check: Inline literal start-string without end-string. (line 53)

WARN: warning: check: Inline interpreted text or phrase reference start-string without end-string. (line 53)

WARN: warning: check: Unexpected possible title overline or transition.
Treating it as ordinary text because it's so short. (line 74)

WARN: warning: check: Unexpected possible title overline or transition.
Treating it as ordinary text because it's so short. (line 81)

error: Please correct your package.
```

File: lib/vsc/install/commontest.py [hunk 1]
```diff
@@ -64,6 +68,13 @@
     except ImportError:
         pass
 
+try:
+    from readme_renderer.rst import render
+    HAS_RENDERER = True
+except ImportError:
+    pass
+
```
Comment (JensTimmerman) [inline lib/vsc/install/commontest.py]: no need for all this trickery, 
just put `tests_require: ['readme_renderer']` in your setup.py

**Comment by boegel** [comment]
@stdweird how do we proceed with this?

**Comment by JensTimmerman** [comment]
@stdweird please fix comments?

**Comment by JensTimmerman** [comment]
follow up in #54

**Pull request closed without merging**